cs.json

```diff
@@ -267,23 +267,23 @@
 			"label-invalid-commercial-license": "Licence je neplatná: ",
 			"button-remove-commercial-license": "Odebrat",
 			"option-catalyst": "Catalyst license",
-			"option-catalyst-desc": "You currently have the {{tier}} Catalyst license. Thanks for your support!",
-			"option-catalyst-desc-no-license": "You don't have a Catalyst license right now. Catalyst license is a one-time donation to support Obsidian development.",
-			"label-app": "App",
-			"label-add-own-language": "See how to add your language to Obsidian."
+			"option-catalyst-desc": "Nyní máte aktivní Catalyst licenci {{tier}}. Děkujeme za podporu!",
+			"option-catalyst-desc-no-license": "Nyní nemáte aktivní Catalyst licenci. Koupí Catalyst licence jednorázově podpoříte vývoj aplikace Obsidian.",
+			"label-app": "Aplikace",
+			"label-add-own-language": "Podívejte se jak přidat váš jazyk do Obsidianu."
```

Comment:
Please don't translate/alter the word "Obsidian"

Reply:
Is it better now? I fixed also some older Obsidian mentions, where the word Obsidian was inflected in other cases.